packages/piece_rope/piece_rope.0.9.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "A data structure for efficiently manipulating strings"
+description:
+  "Supports UTF-8, UTF-16 and UTF-32 for interacting with external systems. Provides extensibility points for you to implement domain-specific functions. Documented and designed to be simple to use."
+maintainer: ["Humza Shahid"]
+authors: ["Humza Shahid"]
+license: "MIT"
+tags: ["text" "piece table" "piece tree" "rope" "unicode"]
+homepage: "https://github.com/hummy123/ocaml-piecerope"
+bug-reports: "https://github.com/hummy123/ocaml-piecerope/issues"
+depends: [
+  "ocaml" {>= "4.13.0"}
+  "dune" {>= "3.6"}
+  "atdgen" {>= "2.11.0"}
+  "yojson" {>= "2.0.2"}
+  "alcotest" {>= "1.7.0"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/hummy123/ocaml-piecerope.git"
```

Comment:
```suggestion
dev-repo: "git+https://github.com/hummy123/ocaml-piecerope.git"
available: arch != "ppc64"
conflicts: [
  "ocaml-option-fp"
  "ocaml-option-flambda"
]
```

Reply:
I am not sure if this conflict is correct, let me ask @kit-ty-kate

Reply:
It looks like this isn't needed anymore. I'm not seeing any issues with frame-pointer, flambda or ppc64

Reply:
Yes, I looked at the size of the offending files (one was 5 MB) and thought it was better to decrease the bundle size and refactor. Thank you both for your help.